src/handbook/customer/marketing/website.md

```diff
@@ -12,19 +12,39 @@ navTitle: Marketing - Website
 
 ## Events Banner
 
-To update the event banner that appears at the top of the Website, you will need two pieces of information:
+The event banner at the top of the website can display more than one event or announcement.
 
-- The title of the Webinar
-- URL that the Webinar Registration is hosted at
+To add or update an event, you'll need to modify the [following file](https://github.com/FlowFuse/website/blob/main/src/_data/events.yaml). The information should be formatted as follows for each banner:
 
-Once you have those, you can update the [following file](https://github.com/FlowFuse/website/blob/main/src/_includes/components/events-banner.njk):
+```
+- type: "Webinar"
+  title: "Deploy FlowFuse on Industrial IoT with NCD.io"
+  buttonText: "Learn more"
+  link: "/webinars/2024/deploy-flowfuse-on-industrial-iot-with-ncd-io/"
+  expire: "2024-05-29T16:00:00Z"
+```
+The `expire` field is used to set the date and time when the event should stop being displayed on the banner. The date and time are set in the ISO 8601 format: `YYYY-MM-DDTHH:MM:SSZ`, and the time is in Coordinated Universal Time (UTC).
+
+For example, `expire: "2024-05-29T15:00:00Z"` means that the event will stop being displayed on the banner at 16:00 UTC on May 29, 2024.
+
+Please note that the website is built once a day at 9:30 AM UTC, and also on Wednesdays and Thursdays at 16:05 UTC. This means that if an event expires at some point during the day, it will still be displayed until the next time the website is built.
```

Comment:
I'd just rebuild it every day at 16:00. This contains a bit too much business logic now tied to webinars.

Reply:
This is the only stuff i can complain about, so I'll merge as is @Yndira-E